third_party/css-escape/css-escape.js

```diff
@@ -1,85 +1,26 @@
 /*! https://mths.be/cssescape v1.5.1 by @mathias | MIT license */
 
 
+var regex = /(\0)|^(-)$|([\x01-\x1f\x7f]|^-?[0-9])|([\x80-\uffff0-9a-zA-Z_-]+)|[^]/g;
```

Comment:
Add a comment that explains what this regex is doing.

Reply:
Done.